src/configuration/evars.md

```diff
@@ -246,6 +246,10 @@ A boolen variable to show UTF-8 characters instead of ANSI.
 
 Enables or disables "fortune" messages displayed at each rizin start.
 
+### cfg.fortunes.file
+
+Rizin has two types for fortunes: tips and fun. Fortunes of the type 'tips' are general tips to help you use Rizin better, whereas the other one prints some lighthearted jokes. You can choose which type of fortune to display, using this variable.
```

Comment:
Actually you can also specify a path (relative or absolute) to a custom fortune file.

Reply:
like `e cfg.fortunes.file /path/to/my/fortunes.txt`

Reply:
Need to update inline docs in Rizin itself as well, because when I read the help message it wasn't clear

Reply:
Alright, I'll add that at the end in a second.
I can help with the inline docs too, if everyone else has other important stuff to do.

Reply:
Another thing, in r2 you could mentioned multiple fortune files, e.g "fun,tips". How does it work now? If it is still like it, need to be mentioned and demonstrated

Reply:
right now there is no multiple file fortunes